### ISSUE TITLE
[Portal] Implement next steps for accepted users

### ISSUE THREAD

**Issue by samderanova** [HackAtUCI/irvinehacks-site-2024#221]
Accepted applicants should be prompted to sign the waiver and RSVP. Note that this is blocked by #219.

Relevant portions we'll need:
- [x] #236
- [x] #237
- [x] #238
- [x] #243
- [x] #253

**Comment by samderanova** [comment]
Duplicate of #219

**Comment by taesungh** [comment]
Closing as all tasks are completed (besides DocuSign handler).
